core/src/avm2/globals/flash/display/stage.rs

```diff
@@ -522,3 +522,23 @@ pub fn set_full_screen_source_rect<'gc>(
     avm2_stub_setter!(activation, "flash.display.Stage", "fullScreenSourceRect");
     Ok(Value::Undefined)
 }
+
+/// Stage.fullScreenHeight's setter
```

Comment:
nit: "getter" in comment - also for the other method

Reply:
Not sure how I managed to miss that 😬